discord/embeds.py

```diff
@@ -15,13 +15,13 @@ def __init__(
         url: str = None,
         type: str = "rich"
     ):
-        self.title = str(title) if title else None
-        self.description = str(description) if description else None
-        self.timestamp = str(timestamp) if timestamp else None
-        self.color = color
-        self.type = type
-        self.url = url
-        self.fields = []
+        self.title: Optional[str] = str(title) if title else None
+        self.description: Optional[str] = str(description) if description else None
+        self.timestamp: Optional[str] = str(timestamp) if timestamp else None
+        self.color: Optional[int] = color
+        self.type: str = type
+        self.url: Optional[str] = url
+        self.fields: list = []
```

Comment:
There are a few more fields for the embed that I included in the new file, try improving on that